tests/segmentation_test.py

```diff
@@ -0,0 +1,165 @@
+import unittest
+import tempfile
+import csv
+import json
+import warnings
+from pathlib import Path
+
+import MDAnalysis as mda
+from MDAnalysis.coordinates.memory import MemoryReader as MDAMemoryReader
+import numpy as np
+
+from lib.segmentation import (
+    chainsaw,
+    merizo,
+    geostas,
+    ParsingError,
+)
+
+class TestSegmentation(unittest.TestCase):
```

Comment:
This is the same test as before, but sadly git can't detect the rename properly. I've kept the original tests and added ones for parsing failures.

I've also had to create an actual PDB to test geostas, because it needs to map atoms to residues now.